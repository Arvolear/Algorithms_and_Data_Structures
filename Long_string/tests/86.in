39059546616727898451267308
1958449936492572