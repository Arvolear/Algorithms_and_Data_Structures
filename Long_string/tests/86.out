39059546618686348387759880
39059546614769448514774736
76496166590959408247393852867429928436176
19944112886